### DISCUSSION TITLE
Licença não entra e repetição dos mesmos videos na lista.

### DISCUSSION THREAD

**Discussion by flavio085** [aclap-dev/video-downloadhelper#826] [License & Purchase requests]
### What's your issue?

Não estou conseguindo registrar a minha licença no firefox e estou com dificuldades para encontrar novos videos na lista que aparece ( sempre aparece os mesmos videos na minha lista )

Comprei a minha licença hoje, 05/06/2024. 

Eu removi a extensão do firefox e depois adicionei de novo, depois disso nao consegui mais registra a minha licença. Por favor, me ajude, preciso baixar meus videos antes do dia 26 desse mês.

<img width="331" alt="A-MINHA-LICENÇA-NAO-ENTRA" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/9ee4e6f1-b1db-4a01-9045-998da72d91c1">
<img width="928" alt="SEMPRE-APARECE-OS-MESMOS-VIDEOS" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/eb2b3c67-af1b-4e87-93a2-a3caaba26321">

### The 10 first digits (only 10) of your license number or email address

3968197f00

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

After you enter your license, you're supposed to click the button I show in this image:

<img width="331" alt="#01" src="https://github.com/aclap-dev/video-downloadhelper/assets/75588964/824bbfbc-40e1-41aa-82e8-b2b88a34224d">

You did the right thing by obscuring your license number with that red blotch.  But there's a neater, easier way to do it:

<img width="375" alt="#02" src="https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0462862a-0773-4d60-b679-fa8f12248653">
<img width="382" alt="#03" src="https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f7654bcc-aa49-4f97-8628-72ddf85fe6e8">

That doesn't unregister your license.  It just removes your license from the display.  Next time you open the settings, you'll see your license is still registered.

**Reply by GrampaWildWilly** [reply]
If you are having problems downloading some videos, you should open a separate problem for that.  Be sure you fill out all the steps in the problem creation dialog.

**Reply by flavio085** [reply]
Eu faço tudo isso. O meu unico problema e´que nao ta registrando minha chave. Ta recusando.
